src/Extensions/Compatability/OmexCompatabilityInitializationException.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.Omex.Extensions.Compatability
+{
+	internal class OmexCompatabilityInitializationException : Exception
+	{
+		internal OmexCompatabilityInitializationException()
+			: base($"Compatability classes were not initialized. Please call AddOmexCompatabilityServices() on HostBuilder")
```

Comment:
Don't use `$` before the string as its not an interpolated string

Reply:
Removed, thank you